src/Command/Setup.php

```diff
@@ -55,15 +55,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $io = new SymfonyStyle($input, $output);
         $io->setDecorated(true);
-        $io->comment(sprintf('Composing your <info>%s</info> environment', (string) $envType));
+        $io->comment(sprintf('Start composing your <info>%s</info> environment', (string) $envType));
 
-        $vars = new AppVendor(
-            $io->ask('Vendor name', null, [AppVendor::class, 'validate']),
-            $io->ask('App name', null, [AppVendor::class, 'validate'])
-        );
+        $host = $io->ask('Host name', 'app', [Host::class, 'validate']);
```

Comment:
Hope the default value did the job but still, if someone has something short enough to be wrapped into parenthesis and added at the end of the question label... I take!

> BTW, maybe we can set the default value to the current directory name instead of simply "app" ?

👍  Looks like a nice idea to me, but I fear it's too much at once for @nervo... :trollface: